crates/tauri-plugin/src/build/mod.rs

```diff
@@ -116,18 +115,23 @@ impl<'a> Builder<'a> {
     }
 
     println!("cargo:rerun-if-changed=permissions");
-    let permissions =
-      acl::build::define_permissions("./permissions/**/*.*", &name, &out_dir, |_| true)?;
+    let permissions = acl::build::define_permissions(
+      &format!("{}/permissions/**/*.*", out_dir.display()),
+      &name,
+      &out_dir,
+      |_| true,
+    )?;
```

Comment:
We have 2 types of permissions, the auto generated ones from the commands (e.g. in your build script, you defined a command `ping`, it will generate `allow-ping` and deny `ping`), and the ones you write yourself, we will need to keep this path

> https://tauri.app/develop/plugins/#permission-files